src/tests/test_scripts/btree_test.py

```diff
@@ -12,16 +12,18 @@
 
 opts, args = getopt.getopt(sys.argv[1:], 'tdlme:',
                            ['test_suits=', 'dirpath=', 'op_list=', 'log_mods=', 'threads=', 'fibers=', 'preload_size=',
-                            'op_list=', 'num_entries=', 'num_iters='])
+                            'op_list=', 'num_entries=', 'num_iters=', 'dev_list=', 'run_time='])
 test_suits = ""
 dirpath = "./"
 op_list = ""
 log_mods = ""
-threads = " --n_threads=10"
-fibers = " --n_fibers=10"
+threads = " --num_threads=10"
+fibers = " --num_fibers=10"
 preload_size = "  --preload_size=2000"
-num_entries = " --num_entries=10000"
-num_iters = " --num_iters=1000000"
+num_entries = " --num_entries=20000"
```

Comment:
num_entries 65k ?

Reply:
OK . we can decide the parameter values later